test/e2e/specs/specs-playwright/wp-plans_purchases-spec.ts

```diff
@@ -0,0 +1,119 @@
+import {
+	DataHelper,
+	LoginFlow,
+	setupHooks,
+	SidebarComponent,
+	PlansPage,
+	IndividualPurchasePage,
+	CartCheckoutPage,
+	BrowserHelper,
+} from '@automattic/calypso-e2e';
+import { Page } from 'playwright';
+
+/**
+ * Constants
+ */
```

Comment:
I used to do this when we didn't have automatic import sorting, but now that we do, I think the `/** constants */` block can go away.